### PR TITLE
sdk/trace: fix grammar errors

### PR DESCRIPTION
**Pull request by kevinburkesegment**
Also use the 'par' program to better align the line breaks in the paragraph doc comments.

### CODE REVIEW

**Comment by MrAlias** [comment]
What spelling mistakes is this addressing? I only see an alternate line break formatting being proposed here.

**Comment by kevinburkesegment** [comment]
Sorry, here.

> The synchronous nature of this SpanProcessor makes it good for testing, debugging, or showing examples of other features

used to be "make" and "feature".

**Comment by MrAlias** [comment]
Can you revert the line break changes and just include the grammatical fixes you want to apply? It will help reviews.

File: sdk/trace/simple_span_processor.go [hunk 1]
```diff
@@ -25,10 +25,10 @@ var _ SpanProcessor = (*simpleSpanProcessor)(nil)
 // send completed spans to the exporter immediately.
 //
 // This SpanProcessor is not recommended for production use. The synchronous
-// nature of this SpanProcessor make it good for testing, debugging, or
-// showing examples of other feature, but it will be slow and have a high
-// computation resource usage overhead. The BatchSpanProcessor is recommended
-// for production use instead.
+// nature of this SpanProcessor makes it good for testing, debugging, or showing
```
Comment (pellared) [inline sdk/trace/simple_span_processor.go:28]: Is this PR changing only `make` to `makes`?

**Review by pellared — APPROVED** [review]

**Comment by pellared** [comment]
@kevinburkesegment Are you able to address https://github.com/open-telemetry/opentelemetry-go/pull/4996#issuecomment-1972220721?

**Review by hanyuancheung — APPROVED** [review]

**Review by dmathieu — APPROVED** [review]

**Pull request merged**